### PR TITLE
For #193: XmlPackage usages

### PR DESCRIPTION
**Pull request by olenagerasimova**
Got rig of string metadata filenames and tags usages with the help of `XmlPackage` enum for #193 and #194.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#197](https://github.com/artipie/rpm-adapter/issues/197) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/74e6de6a-f24c-40a8-885a-7b0d377ba5dc); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/197#issuecomment-632771390))

@victornoel The user @victornoel[/z](https://www.0crat.com/u/victornoel) resigned from [#197](https://github.com/artipie/rpm-adapter/issues/197), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/197#issuecomment-632771390))

@victornoel Job refused in 6 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by olenagerasimova** [comment]
@g4s8 could you ask 0crat assign this PR?

**Comment by 0crat** [comment]
This pull request [#197](https://github.com/artipie/rpm-adapter/issues/197) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/48ed6648-4ec4-46a1-b4dd-d4681783d050); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/artipie/rpm/meta/XmlPackage.java [hunk 1]
```diff
@@ -45,23 +49,44 @@
...
+        public PackageOutput.FileOutput output() throws IOException {
+            return new FilelistsOutput(Files.createTempFile(this.tempPrefix(), XmlPackage.SFX));
+        }
+    };
+
+    /**
+     * File suffix.
+     * @checkstyle ConstantUsageCheck (5 lines)
+     */
+    private static final String SFX = ".xml";
```
Comment (g4s8) [inline src/main/java/com/artipie/rpm/meta/XmlPackage.java:89]: @olenagerasimova what's the problem with this checkstyle `ConstantUsageCheck` issue?  Also, can we use `SUFFIX` name?

Comment (olenagerasimova) [reply src/main/java/com/artipie/rpm/meta/XmlPackage.java]: @g4s8 qulice is complaining 
```
Checkstyle: src/main/java/com/artipie/rpm/meta/XmlPackage.java[88]: Private constant "SUFFIX" is not used (ConstantUsageCheck)
```
seems like a bug, I'll check this one in qulice repo and create a bug of necessary

**Comment by olenagerasimova** [comment]
@g4s8 thanks, I've renamed variable

**Comment by g4s8** [comment]
@olenagerasimova thanks

**Comment by 0crat** [comment]
Job was finished in 5 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**